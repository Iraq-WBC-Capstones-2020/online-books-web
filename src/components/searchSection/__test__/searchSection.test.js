import React from 'react';
import renderer from 'react-test-renderer';

import SearchSection from '../searchSection';

test('searchSection snapshot', () => {
  const tree = renderer.create(<SearchSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
