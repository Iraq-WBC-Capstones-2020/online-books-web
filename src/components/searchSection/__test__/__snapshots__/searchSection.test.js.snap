// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchSection snapshot 1`] = `
<div
  className="container  bg-black p-0 "
  data-test="container"
>
  <div
    className="row justify-content-around"
    data-test="row"
  >
    <div
      className="col-8 col-md-10 col-lg-10 mycolorPink columnSearchBooks p-0 h-12 z-depth-4 "
      data-test="col"
    >
      <form
        className="form-inline searchBox d-flex justify-content-around"
        data-test="form-inline"
        onMouseDown={[Function]}
        onTouchStart={[Function]}
      >
        <div
          className="md-form active-pink"
        >
          
          <input
            aria-disabled={false}
            className="form-control text-black  mb-md-3 form-control "
            data-test="input"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onInput={[Function]}
            placeholder="Find Your Book"
            type="text"
            value=""
          />
          <label
            aria-labelledby=""
            className="active"
            data-error=""
            data-success=""
            id=""
            onClick={[Function]}
          >
             
          </label>
        </div>
        <i
          className="fa fa-search fa-lg text-black hoverCursor pb-3 mb-md-1 "
          data-test="fa"
          onClick={[Function]}
        />
      </form>
    </div>
    <div
      className="col-md-12  text-center pb-3 text-white font-bold "
      data-test="col"
    >
      SEARCH BY
    </div>
    <div
      className="col-6 col-md-6   text-center  px-3 pb-2"
      data-test="col"
    >
      <div
        className="lightBlack h-16 pt-2 text-gray-300 border-orange-500"
        onClick={[Function]}
      >
        <i
          className="fa fa-user-tie fa-lg pb-2 undefined"
          data-test="fa"
        />
        <p
          className="pb-2 textParg undefined"
        >
           Authors
        </p>
      </div>
    </div>
    <div
      className="col-6 col-md-6 text-center px-3 pb-2"
      data-test="col"
    >
      <div
        className="lightBlack text-gray-300 h-16  pt-2"
        onClick={[Function]}
      >
        <i
          className="fa fa-book fa-lg pb-2 textPink"
          data-test="fa"
        />
        <p
          className="pb-2 textParg textPink"
        >
          Books
        </p>
      </div>
    </div>
  </div>
  <div
    className="row mt-2 mb-0"
    data-test="row"
  >
    <div
      className="col-md-12"
      data-test="col"
    >
      <div
        className="text-left lightBlack h-10 mb-1 "
        onClick={[Function]}
      >
        <i
          className="fa fa-circle p-3 sizeBubble textPink"
          data-test="fa"
        />
        <p
          className="parg1 d-inline hoverCursor"
        >
          Popular
        </p>
      </div>
      <div
        className="text-left lightBlack h-10 mb-1 "
        onClick={[Function]}
      >
        <i
          className="fa fa-circle p-3 sizeBubble undefined"
          data-test="fa"
        />
        <p
          className="parg1 d-inline hoverCursor"
        >
          Biographies &Memoirs
        </p>
      </div>
      <div
        className="text-left lightBlack h-10 mb-1 "
        onClick={[Function]}
      >
        <i
          className="fa fa-circle p-3 sizeBubble undefined"
          data-test="fa"
        />
        <p
          className="parg1 d-inline hoverCursor"
        >
          Literature & Fiction
        </p>
      </div>
      <div
        className="text-left lightBlack h-10 mb-1 "
        onClick={[Function]}
      >
        <i
          className="fa fa-circle p-3 sizeBubble undefined"
          data-test="fa"
        />
        <p
          className="parg1 d-inline hoverCursor"
        >
          Business&Investing
        </p>
      </div>
      <div
        className="text-left lightBlack h-10 mb-1 "
        onClick={[Function]}
      >
        <i
          className="fa fa-circle p-3 sizeBubble undefined"
          data-test="fa"
        />
        <p
          className="parg1 d-inline hoverCursor"
        >
          History
        </p>
      </div>
      <div
        className="lightBlack h-10 mb-1"
        md="12"
      />
      <div
        className="lightBlack h-10 mb-1"
        md="12"
      />
      <div
        className="lightBlack h-10 mb-1"
        md="12"
      />
      <div
        className="lightBlack h-10 mb-1"
        md="12"
      />
    </div>
  </div>
</div>
`;
